src/dipdup/config/__init__.py

```diff
@@ -375,6 +375,8 @@ class HasuraConfig:
     :param camel_case: Whether to use camelCase instead of default pascal_case for the field names.
     :param rest: Enable REST API both for autogenerated and custom queries.
     :param http: HTTP connection tunables
+    :param ignore_internal: Whether to ignore internal tables (prefixed with 'dipdup_') when generating the GraphQL API.
```

Comment:
Please, add an entry to `CHANGELOG.md`, then make sure that `make all docs` command passes.

Reply:
yes I've added an entry in the CHANGELOG.md and made sure make all docs command passes successfully